provisioning_templates/snippet/kickstart_networking_setup.erb

```diff
@@ -34,7 +34,6 @@ kind: snippet
 <%- end -%>
 
 <%- @host.managed_interfaces.each do |interface| %>
-<%-   next if !interface.managed? || (interface.subnet.nil? && interface.subnet6.nil?) -%>
```

Comment:
I think this is some leftover from testing, we shouldn't try to configure unmanaged interfaces or interfaces that are not attached to a subnet. All the rest makes sense.